src/build.js

```diff
@@ -28,6 +28,14 @@ module.exports = function build() {
     builder: (yargs) => {
       yargsBuild(yargs);
```

Comment:
why not add to `yargsBuild` ? it is needed also in all other commands, like `up` and `package` and `deploy` ...

Reply:
I saw that `up` extends `build` thus I thought I do not it here.
`deploy` does not seem to run a `build` thus we would not need it here.
`package` needs it, correct, I miss it.
I'll add the arg to `yargsBuild`.

Reply:
ok, found it. `deploy` calls `package` which calls `build`...